addons/crm/models/crm_team.py

```diff
@@ -297,10 +264,11 @@ def _action_assign_leads(self, work_days=1):
         if not self.env.user.has_group('sales_team.group_sale_manager') and not self.env.user.has_group('base.group_system'):
             raise exceptions.UserError(_('Lead/Opportunities automatic assignment is limited to managers or administrators'))
 
-        _logger.info('### START Lead Assignment (%d teams, %d sales persons, %.2f work_days)', len(self), len(self.crm_team_member_ids), work_days)
-        teams_data = self._allocate_leads(work_days=work_days)
+        _logger.info('### START Lead Assignment (%d teams, %d sales persons)', len(self), len(self.crm_team_member_ids))
+        _logger.info('### Force Quota has been enable during the assignment')
```

Comment:
@jeh-odoo enabled* :p Maybe we should also make it conditional :) 

```py
...
if force_quota: 
    _logger.info('### Force Quota has been enabled during the assignment')
...
```

Reply:
Oups... :facepalm: